scripts/generate-package-table-for-readme.js

```diff
@@ -0,0 +1,120 @@
+#! /usr/bin/env node
+
+// This script generates the table of packages you can see in the readme
+
+// Columns to show at the header of the table
+const columns = [
+  'Package',
+  'Version',
+  'Deps',
+  'DevDeps',
+  'CI',
+  'Coverage'
+]
+
+// Headings are a string
+// Arrays are packages. Index 0 is the GitHub repo and index 1 is the npm package
+const rows = [
+  'Repo',
+  ['ipfs/js-ipfs-repo', 'ipfs-repo'],
+
+  'DAG',
+  ['ipld/js-ipld', 'ipld'],
+  ['ipld/js-ipld-dag-pb', 'ipld-dag-pb'],
+  ['ipld/js-ipld-dag-cbor', 'ipld-dag-cbor'],
+
+  'Files',
+  ['ipfs/js-ipfs-unixfs-engine', 'ipfs-unixfs-engine'],
+
+  'Exchange',
+  ['ipfs/js-ipfs-block-service', 'ipfs-block-service'],
+  ['ipfs/js-ipfs-bitswap', 'ipfs-bitswap'],
+
+  'Swarm/libp2p',
```

Comment:
s/Swarm\/libp2p/libp2p/

Reply:
Gotta work on your escaping! Assumed you meant to replace this with just `libp2p` right?